detection_rules/etc/target-branches.yml

```diff
@@ -0,0 +1 @@
+"target_branch_list": [7.16, '8.0', 8.1, 8.2, 8.3]
```

Comment:
as of #2001 we started to reference the [stack-schemas-map](https://github.com/elastic/detection-rules/blob/fd9c9f8abf951b7e759decf350a16dee88445b45/detection_rules/schemas/__init__.py#L258-L268) to centralize which branches are supported.

Reply:
See prior discussions on this topic: https://github.com/elastic/detection-rules/pull/1909#discussion_r899407375 and https://github.com/elastic/detection-rules/pull/1909#discussion_r888959601

Reply:
https://github.com/elastic/detection-rules/pull/1909/files#r903824159